ledger/sandbox/src/main/scala/com/digitalasset/platform/indexer/JdbcIndexer.scala

```diff
@@ -348,6 +348,8 @@ class JdbcIndexer private[indexer] (
     case RejectionReason.Disputed(_) => domain.RejectionReason.Disputed(state.description)
     case RejectionReason.MaximumRecordTimeExceeded =>
       domain.RejectionReason.TimedOut(state.description)
+    case RejectionReason.InvalidLedgerTime(_) =>
```

Comment:
```suggestion
    case _: RejectionReason.InvalidLedgerTime =>
```
@S11001001 Would this prevent an unnecessary `withFilter`?

Reply:
No, you're not in a `for`.